rosdep/python.yaml

```diff
@@ -6095,6 +6095,9 @@ python3-pkg-resources:
   openembedded: [python3-setuptools@openembedded-core]
   rhel: ['python%{python3_pkgversion}-setuptools']
   ubuntu: [python3-pkg-resources]
+python3-progressbar:
+  debian: [python3-progressbar]
+  ubuntu: [python3-progressbar]
```

Comment:
Can you add
- https://packages.gentoo.org/packages/dev-python/progressbar
- https://www.archlinux.org/packages/community/any/python-progressbar/
- https://fedora.pkgs.org/31/fedora-x86_64/python3-progressbar2-3.39.3-3.fc31.noarch.rpm.html
?

Reply:
gentoo, arch and fedora is added in https://github.com/ros/rosdistro/pull/26768/commits/b99dbfb2340b43f608b043b4eb260e3241701073